src/helpers/date.ts

```diff
@@ -0,0 +1,37 @@
+export interface VideoDate {
```

Comment:
Let's use `moment` for all of these things.

Reply:
"Moment.js is a legacy project, now in maintenance mode. In most cases, you should choose a different library." :P